IFComp/lib/IFComp/Schema/Result/User.pm

```diff
@@ -128,8 +128,6 @@ Email doubles as login ID
   is_nullable: 1
   size: 64
 
-User's PayPal account name (usually an email address)
```

Comment:
Is there a way to retain this comment via the db dump? I see there are comments on other fields (e.g. line 70) that weren't removed, so it's probably part of the schema in some way.

Reply:
The comment is part of the database table. If we want to retain it here, we need to add the comment to the production db. I haven't used `COMMENT`s like these before, but it should be the same as modifying a column. Something like:

    ALTER TABLE `user` MODIFY `paypal` `paypal` CHAR(64) NOT NULL COMMENT "User's PayPal account name (usually an email address)";

That said, while DBIC supports adding comments from the database to the schema (when `ifcomp_dump_schema.pl` is used) I don't think it goes the other way. When `ifcomp_deploy_db.sh` is used, the comments aren't added to the database.

So, this will keep happening given the current development process. Until we have a db migration system in place, where developers will hand-write the ALTER statements, we're going to keep losing these comments.

An alternative approach is to manually include the comment as regular POD lower in the file, where it won't be overwritten by `ifcomp_dump_schema.pl`.